packages/compose/index.ts

```diff
@@ -56,6 +56,7 @@ export interface Stamp extends ComposableFactory {
    * @template Obj The type of the object instance being produced by the `Stamp`. or the type of the `Stamp` being created.
    */
   compose: ComposeProperty;
+  [key: string]: any;
```

Comment:
Isn't this already defined in 
> `export interface ComposeProperty extends ComposeMethod, Descriptor, PropertyMap`
?

Why would we add it here too?

Reply:
Another whoops, brute force workaround to something that leaked through. Apologies. Fixed in next commit.